src/test/java/org/broadinstitute/hellbender/utils/tsv/TableUtilsUnitTest.java

```diff
@@ -44,31 +49,51 @@ public void testReader(final TableReader<TestTuple> reader) {
         Assert.assertEquals(actualTuples, TEST_RECORD);
     }
 
-    @Test(dependsOnMethods = "testReader")
-    public void testWriter0() throws IOException {
-        final File testFile = createTempFile("test",".tsv");
-        final TableWriter<TestTuple> writer = TableUtils.writer(testFile,TEST_COLUMNS,
-                (tuple,dataLine) -> {
-                    dataLine.append(tuple.strValue).append(tuple.intValue).append(tuple.dblValue);
-                });
+    private void testWriter0_internal(Path testPath) throws IOException {
+        final TableWriter<TestTuple> writer = TableUtils.writer(testPath, TEST_COLUMNS,
+            (tuple, dataLine) -> {
+                dataLine.append(tuple.strValue).append(tuple.intValue).append(tuple.dblValue);
+            });
         for (final TestTuple tuple : TEST_RECORD) {
             writer.writeRecord(tuple);
         }
         writer.close();
-        final TableReader<TestTuple> reader = TableUtils.reader(testFile,(columns, exceptionFactory) -> {
-            if (!columns.matchesExactly("col1.str","col2.int","col3.dbl"))
-                Assert.fail();
-            return (dataLine) -> new TestTuple(dataLine.get(0),dataLine.getInt(1),dataLine.getDouble(2));
-        });
+        final TableReader<TestTuple> reader = TableUtils
+            .reader(testPath, (columns, exceptionFactory) -> {
+                if (!columns.matchesExactly("col1.str", "col2.int", "col3.dbl"))
+                    Assert.fail();
+                return (dataLine) -> new TestTuple(dataLine.get(0), dataLine.getInt(1),
+                    dataLine.getDouble(2));
+            });
         final TestTuple[] actualTuples = reader.stream().toArray(TestTuple[]::new);
         Assert.assertEquals(actualTuples, TEST_RECORD);
         reader.close();
     }
 
+    @Test(dependsOnMethods = "testReader")
+    public void testWriter0() throws IOException {
+        final Path testPath = createTempPath("testWriter0",".tsv");
+        testWriter0_internal(testPath);
+    }
+
+    /**
+     * Check that TableWriter can accept paths that have no corresponding File
+     * on the local disk.
```

Comment:
Nit perhaps, but can you update this comment as it seems misleading. The purpose of the test is to show that paths on installed NIO file system providers work; the fact that its using an in-memory file system with no on disk representation seems incidental.

Reply:
No problem. The intent was to say "check that this code doesn't try to convert the `Path` back to a `File`" (which you could do if the Path represented a file), I changed the text to be hopefully clearer.